resources/views/components/action.blade.php

```diff
@@ -1,9 +1,32 @@
 {{-- This renderIf method is implemented in every action --}}
 @php
-  $class = 'flex hover:text-blue-600 transition-all duration-300 ease-in-out focus:text-blue-600 active:text-blue-600"'
+    $class = 'flex hover:text-blue-600 transition-all duration-300 ease-in-out focus:text-blue-600 active:text-blue-600"'
 @endphp
 @if ($action->renderIf($item))
-  <a href="#!" title="{{$action->title}}" wire:click.prevent="executeAction('{{ $action->id }}', '{{ $item->getKey() }}', true)" class="{{ $class }}">
-    {{ $slot }}
-  </a>
+
+    <div x-data="{ tooltip: false }" class="cursor-pointer relative z-30 inline-flex"
+         @mousemove.away="tooltip = false"
+    >
+        <button
+                @mousemove="setInterval(function(){tooltip = false}, 2000); tooltip = true"
+                @mouseleave="tooltip = false"
+                wire:click.prevent="executeAction('{{ $action->id }}', '{{ $item->getKey() }}', true)"
+                class="{{ $class }}">
+            {{ $slot }}
+        </button>
+        <div class="relative" x-cloak x-show.transition.origin.top="tooltip"
```

Comment:
I'd move the tooltip to another component wrapping the UI element so we can use the tooltip somewhere else

```html
<x-lv-tooltip tooltip="My tooltip text">
   <!-- Any UI element
   <a />
   <button />
   <img />
   -->
</x-lv-tooltip>
```

What do you think?

Reply:
Nice!